src/quemb/kbe/mf_interfaces/main.py

```diff
@@ -0,0 +1,126 @@
+# ruff: noqa: PLC0415
+
+from collections.abc import Mapping
+from typing import Final, Literal
+
+import numpy as np
+from pyscf.pbc.gto import Cell
+from pyscf.pbc.lib.chkfile import load, load_cell, save, save_cell
+from pyscf.pbc.scf.khf import KRHF
+from typing_extensions import assert_never
+
+from quemb.kbe.mf_interfaces.pyscf_interface import (
+    PYSCF_AVAILABLE,
+    PySCFArgs,
+    create_mf,
+    get_mf_pyscf,
+)
+from quemb.shared.helper import timer
+from quemb.shared.manage_scratch import WorkDir
+from quemb.shared.typing import Matrix, PathLike
+
+SCF_Backends = Literal["pyscf"]
+
+AdditionalArgs = PySCFArgs
+
+AVAILABLE_BACKENDS: Final[Mapping[SCF_Backends, bool]] = {
+    "pyscf": PYSCF_AVAILABLE,
+}
+
+
+@timer.timeit
+def get_mf(
+    cell: Cell,
+    kpts: Matrix[np.floating],
+    *,
+    work_dir: WorkDir | None = None,
+    backend: SCF_Backends = "pyscf",
+    additional_args: AdditionalArgs | None = None,
+) -> KRHF:
+    """
+    Compute the mean-field (SCF) object for a given molecule using the selected backend.
+
+    Supports PySCF as the SCF backend.
+
+    Parameters
+    ----------
+    cell :
+        The cell to perform the SCF calculation on.
+    work_dir :
+        Working directory for external backend calculations (e.g., ORCA).
+        If None, a directory is created based on the environment.
+    backend :
+        The SCF backend to use: "pyscf"
+
+
+    Returns
+    -------
+        The resulting mean-field (KRHF) object from the selected backend.
+    """
+
+    if work_dir is None:
+        work_dir = WorkDir.from_environment(prefix="mf_calculation")
+
+    if backend == "pyscf":
+        if additional_args is None:
+            additional_args = PySCFArgs(
+                density_fit="FFDF"
+            )  # PySCF defaults to FFDF for PBC
```

Comment:
I would rather have
```
additional_args = PySCFArgs()
```

and delegate the definition of defaults to the PySCFArgs class.
Otherwise there are two places where defaults are defined.